src/test/java/com/icthh/xm/uaa/web/rest/AccountResourceIntTest.java

```diff
@@ -28,6 +28,7 @@
 import com.icthh.xm.uaa.service.TenantPropertiesService;
 import com.icthh.xm.uaa.service.TenantRoleService;
 import com.icthh.xm.uaa.service.UserService;
+import com.icthh.xm.uaa.service.account.password.reset.PasswordResetHandlerFactory;
```

Comment:
please add tests for the new functionality.

Reply:
done